dist/carbon.js

```diff
@@ -9460,12 +9632,50 @@ Utils.isRedo = function(event) {
  * @return {boolean} True if it is select all.
  */
 Utils.isSelectAll = function(event) {
-  return !!((event.ctrlKey || event.metaKey) &&
+  return !!((event.ctrlKey || (Utils.isMac() && event.metaKey)) &&
```

Comment:
@assembledadam for the `alt+a` you're seeing on Windows, this is the only thing I can think would introduce this is maybe it is counting the alt key as meta. So I am protecting this with another check for the mac. Let me know if this fixes that issue, if it doesn't I'll have to download a VM to figure out what's happening.